src/algebra/big_operators/fin.lean

```diff
@@ -1,22 +1,108 @@
 /-
-Copyright (c) 2021 Anne Baanen. All rights reserved.
+Copyright (c) 2021 Mario Carneiro, Anne Baanen. All rights reserved.
```

Comment:
The date should be changed as well.

Reply:
Ok.

Reply:
Wait, that's not what I meant  😆 The date should be the oldest of the two, and you should git blame to see who wrote what. Here I believe the first commit is #1903, but there's also #3999 and #9987. So I would suggest
```
Copyright (c) 2020 Yury Kudryashov, Anne Baanen. All rights reserved.
```

Reply:
Thanks!